src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    private static final int NUMBER_BOUND = 100;
```

Comment:
check the documentation of the `nextInt()` method, bound parameter there is exclusive, which means that right now the maximum possible number that we can get is 99

Reply:
Sorry, my mistake, fixed it.